code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -49,6 +49,9 @@
 
 	var/obj/item/radio/integrated/signal/sradio // AI's signaller
 
+	var/pPS = 0 //Do we have pPS installed?
```

Comment:
This variable shouldn't be necessary, right? You can just check the list of purchased software to see if it's owned

I guess you could make the argument that this saves searching a list but the performance hit is negligible I think